Offroad-iOS/Offroad-iOS/Global/Extensions/UIButton+.swift

```diff
@@ -24,5 +24,94 @@ extension UIButton {
 
         setBackgroundImage(backgroundImage, for: state)
     }
+    
+    /// UIButton의 state에 따른 배경색을 설정하는 메서드
+    /// - Parameters:
+    ///   - normal: normal일 때의 font
+    ///   - highlighted: highlighted 됐을 때의 배경색
+    ///   - focused: focused 됐을 떄의 배경색
+    ///   - selected: selected 됐을 때의 배경색
+    ///   - disabled: disabled 됐을 떄의 배경색
+    /// - Description
+    /// normal을 제외한 나머지 매개변수은 매개변수의 값이 nil일 경우 normal의 값이 할당됨.
+    func configureBackgroundColorWhen(
+        normal: UIColor,
+        highlighted: UIColor? = nil,
+        focused: UIColor? = nil,
+        selected: UIColor? = nil,
+        disabled: UIColor? = nil
+    ) {
+        var newConfiguration: UIButton.Configuration
+        if let configuration {
+            newConfiguration = configuration
+        } else {
+            newConfiguration = UIButton.Configuration.filled()
+        }
+        
+        let transformer = UIConfigurationColorTransformer { incomingColor in
+            switch self.state {
+            case .normal:
+                return normal
+            case .highlighted:
+                return highlighted ?? normal
+            case .focused:
+                return focused ?? normal
+            case .selected:
+                return selected ?? normal
+            case .disabled:
+                return disabled ?? .gray
+            default:
+                return self.backgroundColor!
```

Comment:
P3

여기에 강제 언래핑을 사용하면 backgroundColor가 없을 때 문제가 생길 수 있을 것 같습니다. 
`return normal`로 설정하는 건 어떨까요?
```suggestion
                return normal
```

Reply:
좋습니다!
강제언래핑 시에 nil이 나오면 런타임 에러가 발생할 수 있겠네요!
지적 감사합니다!! :)
(이런 경우는 런타임 에러가 발생할 수 있는 사안이므로 P1 리뷰로 다셔도 무방합니다~!)